ui/docs/reference/workers/generic-worker/support-tiers.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Support Tiers
+order: 35
+---
+
+# Support Tiers
+
+While Generic-Worker can run on a wide variety of platforms, the level of support for those platforms varies.
+Support levels are categorized into tiers, each bringing a different set of guarantees.
+This model is based on [Rust's support tiers](https://forge.rust-lang.org/release/platform-support.html).
+
+## Tier 1
+
+Tier one platforms are "guaranteed to work".
+That means that
+* They are tested in Taskcluster's continuous integration and tests must pass for a change to land.
+* Binaries are provided with each Taskcluster release.
+
+The Tier-1 platforms are:
+
+* OSX (GOOS darwin, GOARCH amd64)
+* 64-bit Linux (GOOS linux, GOARCH amd64)
+* 64-bit Linux for ARM (GOOS linux, GOARCH arm64)
+* 32-bit Linux for ARM (GOOS linux, GOARCH arm)
+* 64-bit Windows (GOOS windows, GOARCH amd64)
+* 32-bit Windows (GOOS windows, GOARCH 386)
```

Comment:
I think it is more meaningful to use the binary names since it covers the intended engine (different engines may have different support tiers) and also makes it foolproof to map it back to an actual binary release.

```suggestion
* generic-worker-multiuser-darwin-amd64
* generic-worker-multiuser-linux-amd64
* generic-worker-multiuser-windows-386
* generic-worker-multiuser-windows-amd64
* generic-worker-simple-darwin-amd64
* generic-worker-simple-linux-amd64
* generic-worker-simple-linux-arm
* generic-worker-simple-linux-arm64
```